app/views/users/_form.html.erb

```diff
@@ -1,20 +1,23 @@
 <%= form_tag path, method: method do %>
 <%= label_tag :name %><br>
-<%= text_field_tag :name, @user.name, placeholder: "Name" %><br>
+<%= text_field_tag :name, current_user.name, placeholder: "Name" %><br>
```

Comment:
I like this refactor! Was this more for repeated uses or does this play into security?

Reply:
For the #update action added in this PR, the `current_user` was being updated, not `@user` so this form was throwing errors. To get around this, I changed the value on the form to default to `current_user`. 

#create is still using `@user` (since it needs to create a new user / not just current_user). So in the `new` view, `current_user` is set to `@user` (as a local override). Maybe it would be better practice to set `current_user` to `@user` in the controller?